packages/package_info_plus/package_info_plus_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.0
+
+ - **FEAT**(package_info_plus): add install time ([#3434](https://github.com/fluttercommunity/plus_plugins/issues/3434)). ([0ea0402f](https://github.com/fluttercommunity/plus_plugins/commit/0ea0402f824104c36a806f32e88727801b40ba4c))
```

Comment:
I think something went wrong with versioning as we have the same entry in 3.1.0

Reply:
Indeed, I am not sure why, I corrected that